CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 2.10.0 (April 7, 2022)
+
+NEW:
+
+* Resource `kubernetes_labels` (#692)
+* Resource `kubernetes_annotations` (#692)
+* Resource `kubernetes_config_map_v1_data` (#723)
+* Attribute `rollout` of `wait` (or depricated `wait_for`) block in `kubernetes_manifest` (#1549)
```

Comment:
I would write this as 
```
`kubernetes_manifest` resource: deprecate `wait_for` attribute in favour of `wait` block
```

and put it under improvements rather than new.